evap/results/tests/test_views.py

```diff
@@ -26,7 +26,7 @@ def setUpTestData(cls):
 
 
 class TestResultsSemesterCourseDetailView(ViewTest):
-    url = '/results/semester/2/course/21'
+    url = '/results/semester/1/course/1'
```

Comment:
@karyon can you please take over the tests? i don't even know what this part should be doing and why it worked before with the other course.